cmd/catalyst-node/balancer/balancer.go

```diff
@@ -0,0 +1,297 @@
+package balancer
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"sync"
+
+	serfclient "github.com/hashicorp/serf/client"
+	glog "github.com/magicsong/color-glog"
+)
+
+type BalancerIface interface {
```

Comment:
I think you can just call it `Balancer` and then the implementation `BalancerImpl`. I don't think Go has one convention for this, but the name of the interface is actually used everywhere, so that what should be shorter IMO.

Reply:
Ooh yeah that makes sense, will do